ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/ProcessActionRepository.scala

```diff
@@ -34,13 +35,17 @@ extends BasicRepository with EspTables with CommentActions with ProcessActionRep
 
   import profile.api._
 
+  val PrefixDeployedDeploymentComment = "Deployment: "
```

Comment:
These are used only in repository, please make it private.

Reply:
changed, thx